web/src/components/feature/CalculateFluid.tsx

```diff
@@ -101,7 +100,7 @@ export const CalculateFluid = ({
             <TextField
               id="temperature-input"
               min={-273.15}
-              value={temperature.toString()}
+              placeholder={temperature.toString()}
```

Comment:
What about instead doing
```suggestion
              defaultValue={temperature.toString()}
```
?

Reply:
Yes, that is better. Was looking for it, but couldn't find it.